daemon/cmd/hubble.go

```diff
@@ -208,6 +207,7 @@ func (d *Daemon) launchHubble() {
 		options := []serveroption.Option{
 			serveroption.WithTCPListener(address),
 			serveroption.WithHealthService(),
+			serveroption.WithPeerService(peer.NewService(d.nodeDiscovery.Manager, peerServiceOptions...)),
```

Comment:
Here we could re-use the `peer.Service` used for the local UNIX domain socket right? Also, seems to me that `Stop()` is never called on this new `peer.Service` (same for the one created for the local UNIX domain socket, hence the suggestion to share it).